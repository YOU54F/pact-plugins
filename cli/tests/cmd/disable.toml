bin.name = "pact-plugin-cli"
args = "disable --help"
