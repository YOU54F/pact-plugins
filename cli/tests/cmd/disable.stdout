pact-plugin-cli-disable [CLIVERSION]
Disable a plugin version

USAGE:
    pact-plugin-cli disable <NAME> [VERSION]

ARGS:
    <NAME>       Plugin name
    <VERSION>    Plugin version. Not required if there is only one plugin version

OPTIONS:
    -h, --help    Print help information
